lib/v2/hackmd/profile.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const path = ctx.params.path;
+
+    const response = await got({
+        method: 'get',
+        url: `https://hackmd.io/api/@${path}/overview`,
+    });
+    const data = response.data;
+
+    const profile = data.user || data.team;
+    const notes = data.notes.sort((a, b) => Date.parse(a.createdAt) - b.createdAt);
+
+    ctx.state.data = {
+        title: `${profile.name}'s Profile`,
+        link: `https://hackmd.io/@${path}`,
+        description: `${profile.name}'s profile on HackMD`,
+        item: notes.map((note) => ({
+            title: note.title,
+            description: `<pre>${note.content}</pre>`,
```

Comment:
Since `note.content` contains markdown content, it would be better to render the content

Reply:
I've checked the value of `note.content`, it's pure text without any markup.

Reply:
Please double-check it with the user `@emisjerry`

Reply:
You can check it on the origin website (https://hackmd.io/@emisjerry), it shows the raw markdown in the profile page, not using the rendered content

<img width="659" alt="image" src="https://user-images.githubusercontent.com/172202/211018036-b17dba65-d3e5-4e25-930b-f223aab988d3.png">